docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,199 @@
+| - gevolg       | Een overzicht van Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om details van informatieobjecttypeinformatie op te kunnen vragen moet de consumer weten in welke catalogus het informatieobjecttype is opgenomen. Tevens moet de identficatie van het informatieobjecttype wat uitgevraagd wordt bekend zijn. |
+| - gevolg       | Informatieobjecttypeinformatie zoals opgenomen in de catalogus wordt teruggegeven.             |
+| Gegevens       | INFORMATIEOBJECTTYPE <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/informatieobjecttype_read                                                                                |
+| Samenhang      | - |
+| Bijzonderheden | - |
+
+
+### Resource: eigenschap
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil alle eigenschappen van een zaaktype zoals opgenomen in de catalogus van de organisatie opvragen.             |
+| *Toevoegen*    |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | Nog niet beschikbaar.  |
+| - gevolg       | Nog te bepalen.             |
+| *Raadplegen lijst*   |      |
+| - voorwaarde   | Om eigenschapinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype de eigenschap is opgenomen. |
+| - gevolg       | Een overzicht van eigenschapinformatie van het zaaktype wordt teruggegeven.            |
+| *Raadplegen details*   |      |
+| - voorwaarde   | Om details van eigenschapinformatie op te kunnen vragen moet de consumer weten in welke catalogus en bij welk zaaktype de eigenschap is opgenomen. Tevens moet de identficatie van de eigenschap welke uitgevraagd wordt bekend zijn. |
+| - gevolg       | Detailinformatie van de eigenschap wordt teruggegeven.            |
+| Gegevens       | EIGENSCHAP <br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/eigenschap_read                                                                                |
+| Samenhang      | Een relevant inhoudelijk gegeven dat bj ZAAKen van dit ZAAKTYPE geregistreerd moet kunnen worden en geen standaard kenmerk is van een ZAAK. |
```

Comment:
Wat is de reden om dit te vermelden, en idem hier? Het lijkt mij gekopieerd uit het ImZTC. Dat lijkt me niet zinvol, niet voor niets wordt er naar het ImZTC en het objecttype Eigenschap verwezen. Houd het hier bij functionaliteit.

Reply:
Ok. In dat geval zie ik nog weinig inhoud voor het aspect samenhang en zouden we dat kunnen laten vervallen.

Reply:
Eerst maar 's de documentatie voor enkele API's uitwerken. En dan 's evalueren en waar nodig bijstellen.